windowsapp/msapp-error.html

```diff
@@ -0,0 +1,26 @@
+﻿<!DOCTYPE html>

```

Comment:
These 3 files (msapp-error.*) need to be copied from our existing UWP apps and adapted for Chibi (mostly just changing the colors to fit the theme). It's the page that's shown if there's no internet connection. I can do it in a separate check-in if you want.

Reply:
Lets do it in a separate check-in. I am merging this in.